tokio/src/process/mod.rs

```diff
@@ -1164,7 +1164,11 @@ impl Child {
 
     /// Forces the child to exit.
     ///
-    /// This is equivalent to sending a `SIGKILL` on unix platforms.
+    /// This is equivalent to sending a `SIGKILL` on unix platforms followed by `wait`.
```

Comment:
Please make `wait` a link for easier doc navigation.

Reply:
Added links.